### ISSUE TITLE
Bug: Favorites and Quick Access are empty

### ISSUE THREAD

**Issue by MarioPoier** [files-community/Files#11529]
### Description

When using the 2.4.48.0 version of Files on Windows 11 build 10.0.22000.1574 I have lost the ability to use the Favorites/Quick access. I've tried to uninstall and reinstall. I'm not sure of the location of any cache or configuration files that I could use to reset to default.

### Steps To Reproduce

1. Launch Files
2. Click on Favorites or view Quick Access
![image](https://user-images.githubusercontent.com/16354160/222278753-726df696-2d7a-4b7c-b013-c71bf182e8f9.png)

### Steps to fix

Should be able to view, add, and remove items from Favorites in the left-hand pane (tree view)
Should be able to view, add, and remove items from Quick Items

Basically restore the fresh install functionality.

### Files Version

2.4.48.0

### Windows Version

Windows 11 build 10.0.22000.1574

### Log file

[debug.log](https://github.com/files-community/Files/files/10865722/debug.log)

**Comment by Josh-65** [comment]
Can you send a screenshot of what's pinned in Windows File Explorer

**Comment by MarioPoier** [comment]
![image](https://user-images.githubusercontent.com/16354160/222291127-65689788-d325-4f46-abff-e095bc96fea6.png)

**Comment by MarioPoier** [comment]
I don't know if this helps but it also happens in the current stable release of Files as well.

**Comment by cmactavish** [comment]
Just flagging that I am also having this issue in the current stable version. Let me know if I can provide any useful info.

**Comment by Scouser0001** [comment]
I have exactly the same issue using win10

**Comment by Scouser0001** [comment]
Strike my previous comment. The problem seems to have spontaneously corrected itself without me making any changes to the PC.

**Comment by yaira2** [comment]
@MarioPoier Is this still an issue in the preview version? https://files.community/download/preview

**Comment by MarioPoier** [comment]
Yes it is.
Version 2.4.58.0

Regards,
Mario Poier

**Comment by yaira2** [comment]
Can you share a screenshot of the favorites in File Explorer?

**Comment by MarioPoier** [comment]
Here is the quick access list from File Explorer. I think that is what you are looking for. If it is something else please let me know:
***@***.***

Regards,
Mario Poier

**Comment by yaira2** [comment]
@MarioPoier it doesn't look like the screenshot was uploaded to GitHub, can you try uploading it again?

**Comment by Josh65-2201** [comment]
Should add that email attachments don't work you have to do it directly on GitHub

**Comment by MarioPoier** [comment]
Funnily enough it is now working in v2.4.69. Maybe it was a Windows update???

Regards,
Mario Poier

________________________________

**Comment by yaira2** [comment]
The last few updates of Files fixed a lot of issues so could have been one of them.

**Comment by quimodotcom** [comment]
still an issue with latest release! 
![image](https://github.com/files-community/Files/assets/74259955/22dc76fd-5fa5-4382-b056-1f47f824067b)

**Comment by quimodotcom** [comment]
![image](https://github.com/files-community/Files/assets/74259955/19dbe62f-9927-4ff9-9244-1da5938dea2f)

**Comment by yaira2** [comment]
> still an issue with latest release!

I'm sorry to hear this is still an issue on your device, can you please open a new issue so that we can properly investigate?

**Comment by Scouser0001** [comment]
It does work. I have found that the quick access does not always show when
you first run the program but if you leave it running it will eventually
appear. I have not found a trigger for the behavior and the length of time
it takes to show is quite variable. This does not happen to explorer.
Explorer will show quick acces when Files does not. Once files displays the
quick access it will show even if you stop and start the program during the
day.
